NEWS.md

```diff
@@ -1,5 +1,9 @@
 # remotes 2.0.1.9000 - Development
 
+* `install()` now permits warnings if `R_REMOTES_NO_ERRORS_FROM_WARNINGS` is
```

Comment:
Maybe an  "only"  would be good here. And maybe "now only throws errors for warnings", no?